torch/ao/quantization/backend_config/onednn.py

```diff
@@ -178,6 +178,85 @@ def conv_add_extra_inputs_getter(pattern):
         .set_root_module(nn.Conv2d)
         .set_reference_quantized_module(nnqr.Conv2d))
 
+# (2) Conv2d + Add + Relu
+# conv2d Y
+#   \   /
+#    add
+#     \
+#     relu
+def fuse_conv_add_relu_right(is_qat, relu, add_pattern):
+    add, conv, _ = add_pattern
+    return nni.ConvAddReLU2d(relu, add, conv)
+
+def conv_add_relu_root_node_getter_right(pattern):
+    relu, add_pattern = pattern
+    _, conv, _ = add_pattern
+    return conv
+
+def conv_add_relu_extra_inputs_getter_right(pattern):
+    """ get inputs pattern for extra inputs, inputs for root node
+    are assumed to be copied over from root node to the fused node
+    """
+    relu, add_pattern = pattern
+    _, conv, extra_input = add_pattern
+    return [extra_input]
+
+for add_op in [torch.add, operator.add]:
+    conv_configs.append(
+        BackendPatternConfig()
+            ._set_pattern_complex_format((nn.ReLU, (add_op, nn.Conv2d, MatchAllNode)))
```

Comment:
Always on the right?

Reply:
No, I will add it later. It's still WIP.